src/components/LanguagesList/LanguagesList.html

```diff
@@ -2,7 +2,10 @@
 
   <h1>Languages</h1>
 
-  <ul></ul>
+  <ul>
+    <!--The a tag is only here to provide stylistic consistency-->
+    <li id=add-language-button><a>➕ Add a Language</a><li>
+  </ul>
```

Comment:
Let's use an actual button element for this. The button can sit below the actual `<ul>`, but within the `<nav>`.

```suggestion
  <ul></ul>
  
  <button>Add a Language</button>
```

If you want to include a `+` within the button, let's make sure it's an icon rather than inline text, so that it gets ignored by screen readers since it's just decorative.

We can use [one of these Feather Icons](https://feathericons.com/?query=plus).